dbms/src/Common/HashTable/HashMap.h

```diff
@@ -11,32 +11,28 @@
   */
 
 
-struct NoInitTag {};
+struct NoInitTag
+{
+};
 
 /// A pair that does not initialize the elements, if not needed.
 template <typename First, typename Second>
-class PairNoInit
+struct PairNoInit
```

Comment:
I'd prefer to see this as a separate commit, because it is a mechanical change that touches quite a lot of lines, making it harder to filter out the significant changes.

Reply:
It's Ok. No need to bother.